recipes/cpprestsdk/all/conanfile.py

```diff
@@ -55,17 +55,15 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("boost/1.75.0")
-        self.requires("openssl/1.1.1i")
+        self.requires("boost/1.76.0")
+        self.requires("openssl/1.1.1k")
         if self.options.with_compression:
             self.requires("zlib/1.2.11")
         if self.options.with_websockets:
             self.requires("websocketpp/0.8.2")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = self.name + "-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Comment:
I think this needs a `required_conan_version = ">=1.33.0"` at the top of the file:

Reply:
Yes but we have been so inconsistent with it... I regret not asking for a hook to make sure we apply this as much as possible 

It's missing in most recipes

Reply:
If somebody has the time to create the hook, all recipes can be scanned and fixed.